JWBestPracticeApps/Captions/Captions/AppDelegate.swift

```diff
@@ -16,7 +16,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
         
-        // Add your JW Player license key.
+#warning("Add your JW Player license key.")
```

Comment:
I like the warning, but could you change the `setLicenseKey` line after this to match our other BPA?

Reply:
@s-seibert While I'm at it, should I change all the other BPAs to have the warning rather than the comment?

Reply:
Done